deploy/helm/lunettes/templates/elasticsearch/elasticsearch-clean-job.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: delete-es-index
+  namespace: {{ .Values.namespace }}
+spec:
+  schedule: "{{ .Values.esCleanSchedule }}"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: delete-es-index
+            image: {{ .Values.lunettesImage }}
+            command:
+            - /bin/bash
+            - -c
+            - |
+              # 这里直接执行ConfigMap中的脚本
+              /scripts/delete_index.sh es-cluster-svc.{{ .Values.namespace }}:9200 \
```

Comment:
Just a comment: we may add documentations about the ES indices and the fields for each index for ease of maintainance.